packages/terra-content-container/tests/nightwatch/content-container-spec.js

```diff
@@ -9,20 +9,20 @@ module.exports = {
   },
 
   afterEach: (browser, done) => {
-    screenshot(browser, 'terra-content-container', done);
+    screenshot(browser, '#content-container', done);
   },
 
   'Displays a content container with default props': (browser) => {
     browser.url(`http://localhost:${browser.globals.webpackDevServerPort}/#/tests/content-container-tests/default`);
-    browser.expect.element('.terra-ContentContainer').to.be.present;
-    browser.expect.element('.terra-ContentContainer--filler').to.not.be.present;
-    browser.expect.element('.terra-ContentContainer .terra-ContentContainer-header').to.be.present;
-    browser.expect.element('.terra-ContentContainer .terra-ContentContainer-main').to.be.present;
-    browser.expect.element('.terra-ContentContainer .terra-ContentContainer-main-normalizer').to.be.present;
+    browser.expect.element('#content-container').to.be.present;
+    browser.expect.element('#content-container').to.have.attribute('class').which.not.contains('content-container-fill');
+    browser.expect.element('#content-container div:nth-child(1)').text.to.contain('header');
+    browser.expect.element('#content-container div:nth-child(2)').text.to.contain('content');
   },
+
   'Displays a content container with fill': (browser) => {
     browser
       .url(`http://localhost:${browser.globals.webpackDevServerPort}/#/tests/content-container-tests/fill`)
-      .assert.cssClassPresent('.terra-ContentContainer', 'terra-ContentContainer--fill');
+      .expect.element('#content-container').to.have.attribute('class').which.contains('content-container-fill');
```

Comment:
I think this test should fail if we are using css modules. The css-loader will turn the class name `content-container-fill` into a unique hash. That is done through our webpack configuration [here](https://github.com/cerner/terra-core/blob/master/packages/terra-site/webpack.config.js#L39)